src/ui/Facet/FacetSettings.ts

```diff
@@ -157,7 +157,7 @@ export class FacetSettings extends FacetSort {
     new Popper(this.settingsButton, this.settingsPopup, {
       modifiers: {
         preventOverflow: {
-          boundariesElement: this.facet.root
+          boundariesElement: this.facet.element.parentElement
```

Comment:
Would it be easy to add a unit test for this change?

Reply:
Considering that there is no public reference to this object, sadly no 😢 .